apps/nextjs-website/src/components/organisms/WebinarQuestionsTable/WebinarQuestionsTable.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+import WebinarQuestionRow from '@/components/molecules/WebinarQuestion/WebinarQuestionRow';
+import { updateWebinarQuestion } from '@/lib/webinarApi';
+import { useTranslations } from 'next-intl';
+import { WebinarQuestion } from '@/lib/webinars/webinarQuestions';
+
+type WebinarQuestionsTableProps = {
+  userName: string;
+  questions: WebinarQuestion[];
+  title: string;
+};
+
+const WebinarQuestionsTable = ({
+  userName,
+  questions,
+  title,
+}: WebinarQuestionsTableProps) => {
+  const t = useTranslations('webinar.questionList');
+
+  return (
+    <TableContainer component={Paper} sx={{ marginY: 2 }}>
+      <Table aria-label='simple table'>
+        <TableHead>
+          <TableRow>
+            <TableCell>{t(`title.${title}`)}</TableCell>
```

Comment:
I think here is better to avoid interpolation and directly pass translation from parent component as title